BaikalDataGenerator/Source/main.cpp

```diff
@@ -30,11 +30,48 @@ void Run(const DGenConfig& config)
 
     Render render(config.scene_file, config.width, config.height);
 
-    render.GenerateDataset(config_loader.CamStatesBegin(), config_loader.CamStatesEnd(),
-                           config_loader.LightsBegin(), config_loader.LightsEnd(),
-                           config_loader.SppBegin(), config_loader.SppEnd(),
-                           config.output_dir,
-                           config.gamma_correction);
+    if (config.split_num != 0)
```

Comment:
add the corresponding check to "if" below

Reply:
'split_num' is unsigned